### PR TITLE
Update all embedding description input csv to have 'dcid' and 'sentence' columns

### PR DESCRIPTION
**Pull request by shifucun**
I have ran `run.sh` for all the indexes and confirmed there are no pre-index csv changes.

This PR also fixed/cleaned up the commands and path issue from previous changes.

The original csv files are converted with a script like https://paste.googleplex.com/5485631748964352. Basically merge all the description columns into one column and separated by ";"

### CODE REVIEW

File: tools/nl/embeddings/README.md [hunk 1]
```diff
@@ -80,7 +68,7 @@ variables.
    To generate the `bio_ft` embeddings:
 
    ```bash
-   ./run.sh -c bio data/curated_input/bio,data/curated_input/main data/alternatives/main/*.csv
```
Comment (shifucun) [inline tools/nl/embeddings/README.md:83]: there is no `data/alternatives/main/*.csv` now

File: tools/nl/embeddings/README.md [hunk 2]
```diff
@@ -41,18 +41,6 @@ variables.
 4. Run the command below which will generate a new embeddings csv in
    `gs://datcom-nl-models`. Note down the embeddings file version printed at the end of the run. As of 2023 Q3, there are 3 embeddings used in PROD and the following are the commands to generate each of them:
 
-   To generate the `small` embeddings:
```
Comment (shifucun) [inline tools/nl/embeddings/README.md:44]: remove small embeddings completely

File: tools/nl/embeddings/data/curated_input/sdg/sheets_svs.csv [hunk 3]
```diff
@@ -1,427 +1,427 @@
...
+dcid,sentence
+dc/topic/sdg,Sustainable Development Goals;sdg;sdg;sdg;sdg;sdg Sustainable Development Goals;sustainable development goal Sustainable Development Goals
+dc/topic/sdg_1,No Poverty;sdg 1;sdg 1;sdg1;sdg 1;sdg No Poverty;sustainable development goal No Poverty;poverty rate;poverty
+dc/topic/sdg_1.1,"By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day;sdg 1.1;sdg 11;sdg11;sdg 1 1;sdg By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day;sustainable development goal By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day"
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/sdg/sheets_svs.csv:4]: a comment on how you did these cleanups?

If you used a command line or something, mention it for future reference.

File: tools/nl/embeddings/data/preindex/bio/duplicate_names.csv [hunk 4]
```diff
@@ -1,20 +1,82 @@
 PreferredSV,DroppedSV,DuplicateName
 <-referenceSNPClusterID{typeOf:GeneticVariantGeneAssociation}->geneSymbol,<-geneSymbol{typeOf:GeneticVariantGeneAssociation}->referenceSNPClusterID,GeneticVariantGeneAssociation
 <-diseaseOntologyID{typeOf:DiseaseGeneAssociation}->geneID,<-geneID{typeOf:DiseaseGeneAssociation}->diseaseOntologyID,DiseaseGeneAssociation
-Amount_EconomicActivity_GrossDomesticProduction_Nominal,dc/topic/GDP,GDP
-dc/topic/Mortality,dc/topic/WHOMortality,Mortality
+Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita,Amount_EconomicActivity_GrossDomesticProduction_Nominal_AsAFractionOf_Count_Person,nominal Gross Domestic Product (GDP) per capita
+Annual_Amount_Emissions_Acetaldehyde_SCC_3_IndustrialProcesses,Annual_Amount_Emissions_Acetaldehyde_SCC_23_IndustrialProcesses,amount of acetaldehyde emissions from Industrial Processes
+Annual_Amount_Emissions_Ammonia_SCC_3_IndustrialProcesses,Annual_Amount_Emissions_Ammonia_SCC_23_IndustrialProcesses,amount of ammonia emissions from Industrial Processes
```
Comment (pradh) [inline tools/nl/embeddings/data/preindex/bio/duplicate_names.csv:6]: wonder why there are more duplicate names now?

**Review by pradh — APPROVED** [review]
Much needed, very nice simplification!  Thank you for doing it!

File: tools/nl/embeddings/data/preindex/bio/duplicate_names.csv (hunk 4 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/preindex/bio/duplicate_names.csv:6]: these duplicate names look the same as the duplicate names in the medium index & they are svs that are newly added to sv descriptions for bio in this PR. So my guess is that these are all SVs that did not exist before when bio index was originally created.

**Review by chejennifer — APPROVED** [review]
nice, thanks for the clean up!

File: tools/nl/embeddings/data/curated_input/sdg/sheets_svs.csv (hunk 3 above)
Comment (shifucun) [reply tools/nl/embeddings/data/curated_input/sdg/sheets_svs.csv:4]: Done

**Pull request merged**